spec/operators/catch-spec.ts

```diff
@@ -278,4 +279,83 @@ describe('Observable.prototype.catch', () => {
       done();
     });
   });
+
+  context('fromPromise', () => {
```

Comment:
Happy to change it to `describe`, didn't realize there are no other usages of `context` in the test suite. I personally have used `context` to mean "a different approach to call the same feature" where `describe` is a "different feature". There is no difference in functionality.

```typescript
describe('functionA', () => {
  context('success', () => {...});
  context('error', () => {...});
});
```